projects/helper/tokenMapping.js

```diff
@@ -72,6 +72,15 @@ const fixBalancesTokens = {
     YLDS: { coingeckoId: 'usd-coin', decimals: 0 },
     'pm.sale.pool.3dxq3fk9llvhrqqwhodiap': { coingeckoId: 'usd-coin', decimals: 0 },
   },
+  monad:{
```

Comment:
none of the mappings are required

Reply:
Oh but of the token address are not available on the coreAssets file